deploy/packages/bbp-packages.yaml

```diff
@@ -73,6 +73,7 @@ packages:
       - touchdetector@5.3.1
       - brayns@0.8.0 +opendeck +brion ^ospray@1.7.3
       - brayns@immersive +opendeck +brion +optix
+      - brayns-graphs@0.1.0 ^brayns@1.0.0 +brion
```

Comment:
Why 1.0.0, and everything else requires 0.8.0?

Reply:
not sure if I understand the question. New plugins are working only with latest changes in brayns